docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -335,6 +335,18 @@ This configuration only impacts resources served from the fixed or static URL, `
 
 For more information, see link:https://quarkus.io/blog/path-resolution-in-quarkus/[Path Resolution in Quarkus].
 
+[[add-security-identity-roles]]
+=== Add `SecurityIdentity` roles
+
+Winning role-based policy can add new roles to the `SecurityIdentity` based on the roles `SecurityIdentity` already possess.
+
+[source,properties]
+----
+quarkus.http.auth.policy.admin-policy1.roles.admin=Admin1 <1>
+quarkus.http.auth.permission.roles1.paths=/*
+quarkus.http.auth.permission.roles1.policy=admin-policy1
+----
+<1> Add `Admin1` role to the `SecurityIdentity` with role `admin`.
```

Comment:
Here I propose to start with `Map` the `admin` role to `Admin1` role and note that the identity will have both roles as a result of the mapping

Reply:
made my best attempt